docs/pages/components/Panel.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Panel
+description: A common, versatile pattern used to display content composed of different elements, usually in a collection.
+source: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Panel/Panel.tsx
+---
+
+import {
+  Panel,
+  PanelHeader,
+  PanelContent,
+  Button,
+  Icon
+} from '@deque/cauldron-react';
+
+```js
+import { Panel, PanelHeader, PanelContent } from '@deque/cauldron-react';
+```
+
+## Examples
+
+### Panel Component
+
+At its simplest, a `Panel` is a styled semantic HTML `<section>` element with with content inside.
+
+<Note>
+  If your section needs to have an accessible name, be sure to include an
+  `aria-label` or `aria-labelledby` pointed to the `id` of your label.
+</Note>
+
+```jsx example
+<Panel aria-label="Panel Heading">Panel Content</Panel>
+```
+
+### Panel with Heading (Prop)
+
+To add a heading to a `Panel`, you can use the `heading` prop. One advantage to using this prop over composing
+the elements is it automatically adds an `id` to the heading and `aria-labelledby` to the `Panel` component.
+
+Inspect the below example and you will see that the `Panel` component has an `aria-labelledby` attribute with the
+value of the `id` of the heading.
+
+```jsx example
+<Panel
+  heading={{
+    id: 'panel-heading-prop-example',
+    text: 'Panel Heading',
```

Comment:
This is also duplicated below in the disabled panel heading example. We should choose one to make it unique.